src/test/java/uk/gov/register/presentation/functional/TestEntry.java

```diff
@@ -7,9 +7,17 @@
     public final String itemJson;
     public final Instant entryTimestamp;
 
-    public TestEntry(int entryNumber, String itemJson, Instant entryTimestamp) {
+    private TestEntry(int entryNumber, String itemJson, Instant entryTimestamp) {
         this.entryNumber = entryNumber;
         this.itemJson = itemJson;
         this.entryTimestamp = entryTimestamp;
     }
+
+    public static TestEntry anEntry(int entryNumber, String itemJson){
+        return new TestEntry(entryNumber, itemJson, Instant.now());
```

Comment:
There are two methods. the other method takes the instant as a parameter. Tests which are interested about `entry-timestamp` value can use the other method.